SoftServeCarMobile/src/activity/notifications/NotificationComponent.tsx

```diff
@@ -1,75 +1,27 @@
 import React, { useEffect, useState } from "react";
-import { View, Text, Button, Alert, TouchableOpacity } from "react-native";
+import { View, Text, TouchableOpacity } from "react-native";
 import NotificationStyle from "./NotificationStyle";
-import { Modal } from "react-native";
 import Ionicons from "react-native-vector-icons/Ionicons";
 import "reflect-metadata";
 import { container } from "tsyringe";
 import NotificationsService from "../../../api-service/notifications-service/NotificationsService";
-import { Notification, NotificationType } from "../../../models/Notification";
+import { Notification } from "../../../models/Notification";
 import AvatarComponent from "./AvatarComponent";
+import {JourneyNewApplicant} from "../../components/journey-new-applicant/JourneyNewApplicant";
+import {IComponentsEnum} from "../../common/interfaces/IComponentsEnum";
+import {INotificationProps} from "../../common/interfaces/INotificationProps";
 
 const NotificationComponent = (props: any) => {
-    const [modalTitle, setModalTitle] = useState("default title");
     const notificationService = container.resolve(NotificationsService);
-    const [requestType, setRequestType] = useState("");
+    const [requestType] = useState("");
     const [isModalVisible, setModalVisible] = useState(false);
 
-    const getNotificationDescription = () => {
-        switch (props.item.notificationType) {
-            case NotificationType.PassengerApply:
-                setModalTitle("NEW APPLICANT");
-                setRequestType("asked to join your journey");
-                break;
-
-            case NotificationType.ApplicationApproval:
-                setModalTitle("APPLICATION APPROVAL");
-                setRequestType("approved your application");
-                break;
-
-            case NotificationType.AcceptedInvitation:
-                setModalTitle("NEW PARTICIPANT");
-                setRequestType("accepted your invitation");
-                break;
-
-            case NotificationType.HRMarketingMessage:
-                setModalTitle("HR NEWS");
-                setRequestType("marketing news");
-                break;
-
-            case NotificationType.HRMarketingSurvey:
-                setModalTitle("HR SURVEY");
-                setRequestType("marketing survey");
-                break;
+    let componentsEnumDict: IComponentsEnum<INotificationProps> = {
```

Comment:
Name interfaces without I prefix - it is a convention in typescript.
Also, the suffix Dict is redundant in componentsEnumDict